eth_account/_utils/signing.py

```diff
@@ -103,48 +121,55 @@ def extract_chain_id(raw_v):
         return (chain_id, v_bit + V_OFFSET)
 
 
-def to_standard_signature_bytes(ethereum_signature_bytes):
+def to_standard_signature_bytes(ethereum_signature_bytes: bytes) -> bytes:
     rs = ethereum_signature_bytes[:-1]
     v = to_int(ethereum_signature_bytes[-1])
     standard_v = to_standard_v(v)
     return rs + to_bytes(standard_v)
 
 
-def to_standard_v(enhanced_v):
+def to_standard_v(enhanced_v: int) -> int:
     (_chain, chain_naive_v) = extract_chain_id(enhanced_v)
     v_standard = chain_naive_v - V_OFFSET
     assert v_standard in {0, 1}
     return v_standard
 
 
-def to_eth_v(v_raw, chain_id=None):
+def to_eth_v(v_raw: int, chain_id: Optional[int] = None) -> int:
     if chain_id is None:
         v = v_raw + V_OFFSET
     else:
         v = v_raw + CHAIN_ID_OFFSET + 2 * chain_id
     return v
 
 
-def sign_transaction_hash(account, transaction_hash, chain_id):
+def sign_transaction_hash(
+    account: PrivateKey, transaction_hash: Bytes32, chain_id: Optional[int] = None
+) -> Tuple[int, int, int]:
```

Comment:
I think this can be: 
```suggestion
) -> Tuple[int, Bytes32, Bytes32]:
```
but feel free to ignore if not :)

Reply:
I put a breakpoint in there and the values are `int`s.